### ISSUE TITLE
Initialize Oedt Database Project ConcurrentModificationException

### ISSUE THREAD

**Issue by cverbiest** [omecra/de.omecra.oedt.updatesite#92]
I get following error during startup of PDSOE 12.2

![image](https://user-images.githubusercontent.com/1312691/82822866-c7ad3b00-9ea6-11ea-98a0-a70bc3ab734b.png)
[pdsoe.log](https://github.com/omecra/de.omecra.oedt.updatesite/files/4677733/pdsoe.log)
[pdsoe_config.zip](https://github.com/omecra/de.omecra.oedt.updatesite/files/4677740/pdsoe_config.zip)

**Comment by lneumeier** [comment]
Verified. Note, that Sonarlint has the same problem:

```
java.util.ConcurrentModificationException
	at java.base/java.util.HashMap$HashIterator.nextNode(HashMap.java:1493)
	at java.base/java.util.HashMap$EntryIterator.next(HashMap.java:1526)
	at java.base/java.util.HashMap$EntryIterator.next(HashMap.java:1524)
	at java.base/java.util.Collection.removeIf(Collection.java:544)
	at com.openedge.pdt.project.properties.ConfigFileManager.getConfigFileStatus(ConfigFileManager.java:64)
	at com.openedge.pdt.project.properties.ConfigFileManager.isProjectConfigUsable(ConfigFileManager.java:88)
	at com.openedge.pdt.project.sharedavm.SharedAVM.getProjectsUsingSharedAVM(SharedAVM.java:356)
	at com.openedge.pdt.project.sharedavm.SharedAVM.isSharedAVMProject(SharedAVM.java:326)
	at com.openedge.pdt.project.connection.DatabaseConnectionManager.getProjectDBConnectionProfiles(DatabaseConnectionManager.java:995)
	at com.openedge.pdt.project.connection.DatabaseConnectionManager.loadDatabases(DatabaseConnectionManager.java:2168)
	at com.openedge.pdt.project.connection.DatabaseConnectionManager.loadDatabases(DatabaseConnectionManager.java:2233)
	at com.openedge.pdt.project.connection.DatabaseConnectionManager.getSchemasForProject(DatabaseConnectionManager.java:2141)
	at org.sonarlint.eclipse.pdt.internal.OEProjectConfiguratorExtension.configure(OEProjectConfiguratorExtension.java:139)
	at org.sonarlint.eclipse.core.internal.jobs.AbstractAnalyzeProjectJob.configure(AbstractAnalyzeProjectJob.java:284)
	at org.sonarlint.eclipse.core.internal.jobs.AbstractAnalyzeProjectJob.doRun(AbstractAnalyzeProjectJob.java:164)
	at org.sonarlint.eclipse.core.internal.jobs.AbstractSonarProjectJob.run(AbstractSonarProjectJob.java:45)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
```

**Comment by cverbiest** [comment]
Thank you for pointing that out,I created https://github.com/Riverside-Software/sonar-openedge/issues/799 for the Sonarlint issue

**Comment by gquerret** [comment]
@lneumeier Did you already report to Progress ?

**Comment by lneumeier** [comment]
@gquerret 

No, I tried it several times for other issues with no luck.

@cverbiest 

This issue should be fixed with OEDT 9.3.3.
